webrender/src/frame.rs

```diff
@@ -319,6 +319,31 @@ impl Frame {
                 continue;
             }
 
+            let mut delta:Point2D<f32> = match scroll_location {
+                ScrollLocation::Delta(delta) => delta,
+                ScrollLocation::Start => {
+                    if layer.scrolling.offset.y.round() == 0.0 {
+                        // Nothing to do.
+                        return false;
```

Comment:
okay, sure :)

I was thinking something like this would work for these blocks:

```rust 
ScrollLocation::Start => {
  if layer.scrolling.offset.y.round() == 0.0 {
    // Nothing to do on this layer.
    continue;
  }

  layer.scrolling.offset.y = 0.0;
  scrolled_a_layer = true;
  continue;
},
```

What do you think? I have put these changes in and updated with latest WR master in the latest push